starter-code/src/viking.js

```diff
@@ -1,11 +1,111 @@
 // Soldier
-function Soldier() {}
+function Soldier(health, strength){
+  this.health = health;
+  this.strength = strength;
+};
+
+Soldier.prototype.attack = function(){
+  return this.strength;
+};
+
+Soldier.prototype.receiveDamage = function(damage){
+  this.health -= damage;
+};
+
+// Test to create a new Soldier
+// var newSoldier = new Soldier("10","2");
+// console.log(newSoldier);
 
 // Viking
-function Viking() {}
+function Viking(name, health, strength){
+  Soldier.call(this, health, strength);
+  this.name = name;
+  // this.battlecry = "Odin Owns You All!";
+  
+  Viking.prototype = Object.create(Soldier.prototype);
+
+  Viking.prototype.receiveDamage = function (damage){
+    result = "";
+    this.health -= damage;
+
+    if(this.health > 0){
+      return this.name + " has recieved " + damage +  " points of damage";
+    } else{
+      return this.name + " has died in act of combat";
+    };
+  };
+};
+
+  Viking.prototype.battleCry = function(){
+    return "Odin Owns You All!";
+  };
+ 
+
+  // Viking.prototype.constructor = Viking;
+
+// Test to create a new Viking
+// var newViking = new Viking("Mike", "10", "2");
+// console.log(newViking); 
+
+// // Saxon
+function Saxon(health, strength){
+  Soldier.call(this, health, strength);
+
+  Saxon.prototype = Object.create(Soldier.prototype);
+
+  Saxon.prototype.receiveDamage = function(damage){
+
+    if(health > 0){
```

Comment:
`if(health > 0){` should have been `this.health` . and you should deal the damage before confirming if health > 0 not after.  `this.health -= damage` should be above the if statement to confirm if after the damage has been dealt whether the Saxon is dead or not.